tests/plugins/section/test_plugin.py

```diff
@@ -1,21 +1,20 @@
 """
 Section plugin tests
 """
-from django.db import IntegrityError
-from django.test import TestCase
-from django.test.client import RequestFactory
+from django.db import IntegrityError, transaction
 
 from cms.api import add_plugin
 from cms.models import Placeholder
-from cms.plugin_rendering import ContentRenderer
 
+from richie.apps.core.tests.utils import CMSPluginTestCase
 from richie.plugins.section.cms_plugins import SectionPlugin
 from richie.plugins.section.factories import SectionFactory
 
 
-class SectionTests(TestCase):
+class SectionTests(CMSPluginTestCase):
     """Section plugin tests case"""
 
+    @transaction.atomic
```

Comment:
Worth to mention i don't really know why i needed to turn this test in atomic version, else it was breaking every other test following it. Troubling since another similar test in Person plugin does not need it.

Reply:
then this is not a solution. When a test is breaking the following tests, it means it is doing something wrong. Like modifying the active language outside of a "with" clause or something like this?

Reply:
Stated than this is an unknow issue provoked by CMSTestCase mixin.